flink-formats/flink-parquet/pom.xml

```diff
@@ -161,6 +186,48 @@ under the License.
 				</executions>
 			</plugin>
 
+			<!-- Geneate Test class from protobuf schema -->
+			<plugin>
+				<groupId>org.xolstice.maven.plugins</groupId>
```

Comment:
Protobuf need this?

Reply:
This plugin is used to download the corresponding protoc and do the code generation for the .proto file.